camel/models/litellm_model.py

```diff
@@ -0,0 +1,98 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from typing import Any, Dict, List, Optional, Union
+
+from litellm import completion
+from litellm.utils import (
+    CustomStreamWrapper,
+    ModelResponse,
+)
+
+from camel.configs import LITELLM_API_PARAMS
+from camel.messages import OpenAIMessage
+from camel.models import BaseModelBackend
+from camel.types import ModelType
+from camel.utils import BaseTokenCounter, LiteLLMTokenCounter, api_key_required
+
+
+class LiteLLMModel(BaseModelBackend):
+    r"""LiteLLM API in a unified BaseModelBackend interface."""
+
+    def __init__(
+        self, model_type: ModelType, model_config_dict: Dict[str, Any]
```

Comment:
let's set `model_type` as `str` since for open source models it's hard to maintain all types in enum

Reply:
You are right, the change has been made.👌